docs/tutorial/source/tutorial-4.py

```diff
@@ -91,13 +91,25 @@ def draw_nostrils(self):
             self.canvas.move_to(45, 145)
             self.canvas.bezier_curve_to(51, 123, 96, 123, 102, 145)
 
+    def draw_text(self):
+        x = 32
+        y = 175
+        font = toga.Font(family='sans-serif', size=20)
+        width, height = self.canvas.measure_text('Tiberius', font)
```

Comment:
Random thought - is this a function of the *canvas*, or the *font*? I can think of situations where you might want to know how big a piece of text is, but *don't* have a canvas on which to measure it.